_config.yml

```diff
@@ -35,3 +35,32 @@ kramdown:
   input: GFM
   hard_wrap: false
   parse_block_html: true
+
+collections:
```

Comment:
We no longer need this collection for the `release_notes`.
I think this code remained as a result of the merge.